Fitness/Views/ProBio/ProBioLinksCell.swift

```diff
@@ -0,0 +1,135 @@
+//
+//  ProBioLinksCell.swift
+//  Fitness
+//
+//  Created by Yana Sang on 5/20/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class ProBioLinksCell: UICollectionViewCell {
+
+    // MARK: - Private view vars
+    private var linksCollectionView: UICollectionView!
+    private let linksTitleLabel = UILabel()
+    private let linksInterItemSpacing = 6
```

Comment:
Move this to dataVars or within setupView()

Reply:
moved to data vars because its used in two functions